pkg/pillar/types/zedroutertypes.go

```diff
@@ -294,8 +294,9 @@ type NetworkPortStatus struct {
 	NetworkXObjectConfig
 	AddrInfoList []AddrInfo
 	ProxyConfig
-	Error     string
-	ErrorTime time.Time
+	Error            string
+	ErrorTime        time.Time
+	CloudUnReachable bool
```

Comment:
Cosmetic - It is more readable to make the Variable a positive one ( CloudReachable ) as opposed to "UnReachable" ? Generally keeps the logic more straight forward..

Reply:
@kalyan-nidumolu That doesn't work since the language doesn't support implicit initializers AFAIK.
An explicit initializer function isn't helpful when there are nested data structures.